nemo_skills/pipeline/check_contamination.py

```diff
@@ -126,10 +135,14 @@ def check_contamination(
     get_random_port = server_gpus != 8 and not exclusive
 
     cluster_config = get_cluster_config(cluster, config_dir)
-    check_if_mounted(cluster_config, input_file)
-    check_if_mounted(cluster_config, output_file)
-    if log_dir:
-        check_if_mounted(cluster_config, log_dir)
+    cluster_config = resolve_mount_paths(cluster_config, mount_paths)
```

Comment:
maybe make it a part of get_cluster_config (if mount_paths specified we resolve them)?

Reply:
This needs to be separate. Otherwise we have to go back and update all get_cluster_config with optional arg resolve mount paths. And on top of that, in python mode, you should call get cluster config for "base config" and then run custom dicts / str of resolve mount paths per task you're executing